hoodie-client/src/main/java/com/uber/hoodie/config/HoodieMemoryConfig.java

```diff
@@ -45,6 +45,9 @@
   public static final String MAX_MEMORY_FOR_MERGE_PROP = "hoodie.memory.merge.max.size";
   // Property to set the max memory for compaction
   public static final String MAX_MEMORY_FOR_COMPACTION_PROP = "hoodie.memory.compaction.max.size";
+  // Property to set the max memory for hdfs inputstream buffer size
+  public static final String MAX_DFS_STREAM_BUFFER_SIZE_PROP = "hoodie.memory.hdfs.buffer.max.size";
```

Comment:
hoodie.memory.dfs.buffer.max.size?

Reply:
ahh, apologies for the oversight, will rename correctly